intyg-proxy-service/integration-api/src/main/java/se/inera/intyg/intygproxyservice/integration/api/employee/GetEmployeeRequest.java

```diff
@@ -0,0 +1,12 @@
+package se.inera.intyg.intygproxyservice.integration.api.employee;
+
+import lombok.Builder;
+import lombok.Value;
+
+@Value
+@Builder
+public class GetEmployeeRequest {
```

Comment:
In minaintyg we added integration to the naming of requests, response and service. This is to make it easily extinguishable from app service. In this case it would be GetEmployeeIntegrationRequest.